src/octopeer-github/content/ElementEventBinding/ElementEventBindingData.ts

```diff
@@ -37,11 +37,45 @@ const elementEventBindingDataList: ElementEventBindingData[] = [
         name: "mouseleave",
     },
     {
-        eventID: EventID.SCROLL_INTO_VIEW,
-        name: "scrollintoview",
-    },
-    {
-        eventID: EventID.SCROLL_OUT_OF_VIEW,
-        name: "scrolloutofview",
+        addDOMEvent: (elementSelectionBehaviour: ElementSelectionBehaviour) => {
```

Comment:
I know that `handleScrollEvent` _should_ be inside the scope of `addDOMEvent` because it depends on scope variable `wasInScope`. Maybe the functions `isBetween`until `isInScope`can be moved somewhere, but since they're only used here we were unsure where to move them. If you have any suggestions, we can consider them. :)